src/app/pages/jails/jail-form/jail-form.component.ts

```diff
@@ -861,18 +861,19 @@ export class JailFormComponent implements OnInit, AfterViewInit {
     protected networkService: NetworkService) { }
 
   getReleaseAndInterface() {
-    if (this.plugin !== undefined) {
-      this.jailFromService.getInterface().subscribe(
-        (res) => {
-          for (let i in res) {
-            this.interfaces.vnetDisabled.push({ label: res[i].name, value: res[i].name });
-          }
-        },
-        (res) => {
-          new EntityUtils().handleWSError(this, res, this.dialogService);
+    this.jailService.getInterfaceChoice().subscribe(
+      (res) => {
+        for (const i in res) {
```

Comment:
The key of jail.interface_choices should be the value, the label should be the value of the object because it can return values like:

{"igb0": "igb0: description here"}

Reply:
get it. thanks @william-gr